lib/model/internal_link.dart

```diff
@@ -59,7 +59,7 @@ String? decodeHashComponent(String str) {
 Uri narrowLink(PerAccountStore store, Narrow narrow, {int? nearMessageId}) {
```

Comment:
Some nits on commit message:
- `updates` -> `Updates`
- `...fragment identifier This...` -> `...fragment identifier. This...`
- `likified` -> `linkified`

and an empty space between the paragraph and `Fixes: #845`.